src/app/fdctl/run/tiles/fd_quic.c

```diff
@@ -616,23 +545,15 @@ privileged_init( fd_topo_t *      topo,
   ctx->ip = fd_ip_join( fd_ip_new( FD_SCRATCH_ALLOC_APPEND( l, fd_ip_align(), fd_ip_footprint( 256UL, 256UL ) ), 256UL, 256UL ) );
   if( FD_UNLIKELY( !ctx->ip ) ) FD_LOG_ERR(( "fd_ip_join failed" ));
 
+  fd_quic_limits_t limits = quic_limits( tile );
+  void * quic_mem  = FD_SCRATCH_ALLOC_APPEND( l, fd_quic_align(), fd_quic_footprint( &limits ) );
+  fd_quic_t * quic = fd_quic_join( fd_quic_new( quic_mem, &limits, ctx->ip ) );
+  if( FD_UNLIKELY( 32UL!=getrandom( quic->config.identity_key, 32UL, 0 ) ) )
```

Comment:
Can we just pull the correct key here? See `identity_key_path` in `topology.h`, you can just copy how it's used by, eg, the shred tile `fd_shred.c`.